packages/arb-token-bridge-ui/tests/e2e/specs/withdrawERC20.cy.ts

```diff
@@ -9,244 +9,259 @@ import {
-      cy.findByLabelText('WETH balance amount on parentChain')
-        .should('be.visible')
-        .its('text')
-        .should('not.eq', l1ERC20bal)
-    })
+        cy.findByText(
+          `${formatAmount(ERC20AmountToSend, {
+            symbol: testCase.symbol
+          })}`
+        ).should('be.visible')
 
-    it('should withdraw ERC-20 to custom destination address successfully', () => {
-      const ERC20AmountToSend = Number((Math.random() * 0.001).toFixed(5)) // randomize the amount to be sure that previous transactions are not checked in e2e
+        cy.findByLabelText('Close side panel').click()
 
-      cy.login({ networkType: 'childChain' })
-      context('should add a new token', () => {
         cy.searchAndSelectToken({
-          tokenName: 'WETH',
-          tokenAddress: l2WethAddress
+          tokenName: testCase.symbol,
+          tokenAddress: testCase.l2Address
         })
-      })
-
-      context('should show summary', () => {
-        cy.typeAmount(ERC20AmountToSend)
-        cy.findGasFeeSummary(zeroToLessThanOneETH)
-        cy.findGasFeeForChain(getL2NetworkName(), zeroToLessThanOneETH)
-        cy.findGasFeeForChain(
-          new RegExp(
-            `You'll have to pay ${getL1NetworkName()} gas fee upon claiming.`,
-            'i'
-          )
-        )
-      })
 
-      context('should fill custom destination address successfully', () => {
-        cy.fillCustomDestinationAddress()
-      })
-
-      context('should show clickable withdraw button', () => {
-        cy.findMoveFundsButton().click()
+        // the balance on the destination chain should not be the same as before
+        cy.findByLabelText(`${testCase.symbol} balance amount on parentChain`)
+          .should('be.visible')
+          .its('text')
+          .should('not.eq', l1ERC20bal)
       })
 
-      context('should initiate withdrawal successfully', () => {
-        cy.findByText(/Arbitrum’s bridge/i).should('be.visible')
+      it(`should withdraw ${tokenType} to custom destination address successfully`, () => {
+        const ERC20AmountToSend = Number((Math.random() * 0.001).toFixed(5)) // randomize the amount to be sure that previous transactions are not checked in e2e
 
-        // the Continue withdrawal button should be disabled at first
-        cy.findByRole('button', {
-          name: /Continue/i
-        }).should('be.disabled')
-
-        cy.findByRole('switch', {
-          name: /before I can claim my funds/i
+        cy.login({ networkType: 'childChain' })
+        context('should add a new token', () => {
+          cy.searchAndSelectToken({
+            tokenName: testCase.symbol,
+            tokenAddress: testCase.l2Address
+          })
         })
-          .should('be.visible')
-          .click()
 
-        cy.findByRole('switch', {
-          name: /after claiming my funds/i
+        context('should show summary', () => {
```

Comment:
https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests#Test-Structure

We should probably stick to only one

Reply:
Should be covered in FS-757